src/openrct2/peep/peep.c

```diff
@@ -9784,7 +9784,7 @@ sint32 peep_pathfind_choose_direction(sint16 x, sint16 y, uint8 z, rct_peep *pee
 }
 
 /**
- * Gets the nearest park entrance relative to point, by using Manhatten distance.
+ * Gets the nearest park entrance relative to point, by using Manhatten Distance.
```

Comment:
To be fair, LinusU suggested "Manhattan Distance" with a capital D. Don't be too harsh.